pkgs/applications/misc/peaclock/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, cmake, libpthreadstubs, icu }:
+
+stdenv.mkDerivation rec {
+  pname = "peaclock";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "octobanana";
+    repo = pname;
+    rev = version;
+    sha256 = "1582vgslhpgbvcd7ipgf1d1razrvgpq1f93q069yr2bbk6xn8i16";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ libpthreadstubs icu ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "A clock, timer, and stopwatch for the terminal.";
```

Comment:
`meta.description` should not end with a period, according to the [contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes).

Reply:
Removed the period.